test/test_arraycontext.py

```diff
@@ -219,9 +222,12 @@ def assert_close_to_numpy_in_containers(actx, op, args):
     obj_array_args = [
             make_obj_array([arg]) if isinstance(arg, DOFArray) else arg
             for arg in dofarray_args]
-    obj_array_result = actx.to_numpy(op(actx.np, *obj_array_args)[0][0])
 
-    assert np.allclose(obj_array_result, ref_result)
+    obj_array_result = op(actx.np, *obj_array_args)
+    if isinstance(obj_array_result, np.ndarray):
+        obj_array_result = obj_array_result[0][0]
+
+    assert np.allclose(actx.to_numpy(obj_array_result), ref_result)
```

Comment:
Do any of the tests exercise complex-valued inputs?

Reply:
Extended some of the tests to also check complex inputs in 5d67940.